src/go/k8s/config/rbac/bases/operator/role.yaml

```diff
@@ -272,6 +272,13 @@ rules:
   - get
   - patch
   - update
+- apiGroups:
+  - scheduling.k8s.io
+  resources:
+  - priorityclasses
+  verbs:
```

Comment:
I normally add watch verb to the list below. Yet the code that checks storage class is within webook, so it might not have proper client cache configured

Reply:
Added.